lsif/src/dump-manager/settings.ts

```diff
@@ -18,3 +18,15 @@ export const DOCUMENT_CACHE_CAPACITY = readEnvInt('DOCUMENT_CACHE_CAPACITY', 102
 
 /** The maximum number of result chunks that can be held in memory at once. */
 export const RESULT_CHUNK_CACHE_CAPACITY = readEnvInt('RESULT_CHUNK_CACHE_CAPACITY', 1024 * 1024 * 1024)
+
+/** The maximum rate that the server will send upload payloads. */
+export const MAXIMUM_SERVE_BITS_PER_SECOND = readEnvInt('MAXIMUM_SERVE_BITS_PER_SECOND', 1024 * 1024 * 1024 * 10)
```

Comment:
```suggestion
export const MAXIMUM_SERVE_BITS_PER_SECOND = readEnvInt('MAXIMUM_SERVE_BITS_PER_SECOND', 1024 * 1024 * 1024 * 10)  // 10 Gib/sec
```

That seems unrealistically high, is this aggregated over all queries?

Also: Is there some reason this is bits rather than bytes?

Reply:
Oops I may have fudged this and am already using it as bytes :o

This number was pulled from @slimsag's suggestion that we cap uploads to something high so we don't take down a the docker network under high load. Up for discussion around this feature (high level and specifics).

Reply:
High makes sense—we don't want to cap more than necessary to keep the host from falling over. But I would be surprised if any container network could push anywhere close to 10 Gib/sec., at which point why bother having the cap at all? 

10 Mib/sec certainly seems plausible, though.

Reply:
> That seems unrealistically high, is this aggregated over all queries?

Yes, it's a global limit for all inbound/outbound data transfer into this router.